### ISSUE TITLE
URL not stripped on paste

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#1565]
## Steps to Reproduce

1. Create a new thought
2. Paste any url from a browser (e.g. https://stackoverflow.com/)

## Current Behavior

The link appears underlined and blue, which are not allowed in **em**. The formatting is stripped only after the cursor changes. 

## Expected Behavior

The formatting of URLs should be stripped immediately on paste.

**Comment by minxie97** [comment]
Hello, do you mind if I took a crack at this?

**Comment by raineorshine** [comment]
Sure!

Here is where `onPaste` is handled: https://github.com/cybersemics/em/blob/dev/src/components/Editable.tsx#L545

HTML and formatting are typically stripped here: https://github.com/cybersemics/em/blob/dev/src/util/strip.ts

The `Editable` component is very complex and not very well organized, so please accept my apologies!

Let me know if you have any questions.

**Comment by aboqasem** [comment]
Hi @raineorshine, I tried to reproduce this issue but I could not. Could you please elaborate more?

**Comment by raineorshine** [comment]
I tested it again and it looks like this is Safari specific. In mobile Safari, if you hit the "Share" button and choose "Copy" it will copy the correct data to reproduce.

The weird thing is that `e.clipboardData.getData` is empty with both `text/plain` and `text/html`.

Actually, I stumbled upon a solution. It looks like when using the share button, Mobile Safari copies it as `text/uri-list`. If we read `text/uri-list` and import it as plain text then it works.
